### PR TITLE
Remove research recruitment banner

### PR DESCRIPTION
**Pull request by lesleyjanenorton**

### CODE REVIEW

File: public/js/monitor.js [hunk 1]
```diff
@@ -441,40 +427,15 @@ function resizeDashboardMargin() {
...
-      sendRecruitmentPing("Engage", false, "beacon");
-    });
-  }
-
   const dropDownMenu = document.querySelector(".mobile-nav.show-mobile");
   dropDownMenu.addEventListener("click", () => toggleDropDownMenu(dropDownMenu));
 
 
 
+  const acceptedLanguages = navigator.languages;
```
Comment (maxxcrawford) [inline public/js/monitor.js:438]: @lesleyjanenorton Added this back as it's in use below.

Comment (lesleyjanenorton) [reply public/js/monitor.js:438]: could be:
```
const fxaCheckbox = document.getElementById("fxaCheckbox");
if (!acceptedFirstLanguageIsEnglish && fxaCheckbox) {
  fxaCheckbox.style.display = "none" 
}

**Comment by lesleyjanenorton** [comment]
@maxxcrawford your updates look good - no problems on my end running locally.

**Comment by groovecoder** [comment]
Code LGTM. Local spot-checks work.

**Pull request merged**